spec/ics-007-tendermint-client/README.md

```diff
@@ -89,12 +93,18 @@ interface Evidence {
 Tendermint client initialisation requires a (subjectively chosen) latest consensus state, including the full validator set.
 
 ```typescript
-function initialise(consensusState: ConsensusState, validatorSet: List<Pair<Address, uint64>>, height: uint64): ClientState {
-  return ClientState{
-    validatorSet,
-    latestHeight: height,
-    pastHeaders: Map.singleton(latestHeight, consensusState)
-  }
+function initialise(
+  consensusState: ConsensusState, validatorSet: List<Pair<Address, uint64>>,
+  height: uint64, trustingPeriod: uint64, unbondingPeriod: uint64): ClientState {
+  assert(trustingPeriod < unbondingPeriod)
```

Comment:
Hmm. For safety, it should be strictly less than, as if they are equal at the very end of the trusting period there won't be enough time to submit evidence (since blocks aren't instant).

Reply:
@melekes is there any restriction to this check? ^^

Reply:
@cwgoes is right (it must be strictly less) minus some delta for evidence propagation (network delay + how often blocks are created). e.g. if blocks are created once a day, then delta should be 2 days.